backend/controllers/feedbackController.js

```diff
@@ -0,0 +1,23 @@
+const asyncHandler = require("express-async-handler");
+const Feedback = require("../models/feedbackModel");
+const bcrypt = require("bcryptjs");
```

Comment:
The imports from bcrypt (line 3 to line 8) onwards aren't used. Lets remove them

Reply:
Fixed: Changes are reflected in recent commits